### ISSUE TITLE
support the full palette of the keynote version

### ISSUE THREAD

**Issue by k4b7** [xymostech/markdown-ka-presentation#1]
The colors are blue, teal, green, gold, and red.  I'll dig up the actual hex colors for all of the shades of those colors later.  I was thinking that to actually use the colors we do something like:

```
~~~s(blue)
This is a blue section slide
~~~ss(teal)
This is a teal sub-section slide
~~~(green)
This is a green slide
...
```

Since each letter is different we could also shorten this to:

```
~~~s(b)
This is a blue section slide
...
```

**Comment by xymostech** [comment]
That sounds great to me! I ended up not needing any of the other colors which is why I didn't add them.

Just curious, what sort of presentation are you trying to build using this? I'm excited for someone else to try it out and see how well it works for them. :)

**Comment by k4b7** [comment]
I'm working on a presentation that will give an overview of linting at KA and then talk about writing custom linters using eslint.  I'm prepping for a talk at js-montreal next month.
